cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,10 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
+		set:  SetBool,
```

Comment:
Not sure which ones you are referring to precisely, but afaicr this is all inherited